website/content/docs/security/acl/acl-rules.mdx

```diff
@@ -297,7 +394,86 @@ keyring = "write"
 There's only one keyring policy allowed per rule set, and its value is set to one of the policy
 dispositions. In the example above, the keyring may be read and updated.
 
-#### Node Rules
+### Mesh Rules
+
+The `mesh` resource controls access to ingress gateways, terminating gateways, and mesh configuration entries. 
+
+See [Admin Partition Rules](#admin-partition-rules) for an example rule that uses the `mesh` resource. 
```

Comment:
It is nice to link to a more complete example, but for clarify I think it might be good to have a minimal example here first before this the "see Admin Partition Rules..." line

```hcl
mesh = "write"
```